pyogrio/geopandas.py

```diff
@@ -19,6 +20,28 @@ def _stringify_path(path):
     return path
 
 
+def _try_parse_datetime(ser):
+    import pandas as pd  # only called in a block where pandas is known to be installed
+
+    pandas_ge_20 = Version(pd.__version__) >= Version("2.0.0")
```

Comment:
a global variable toward the top of `geopandas.py` seems reasonable; since this is pretty isolated, I don't think we need another file to manage that.